NOTICE.txt

```diff
@@ -1,753 +0,0 @@
-NOTICES AND INFORMATION
```

Comment:
Your PR is accidentally deleting this file.  You'll need to add it back.

Reply:
Nevermind. It looks like GitHub showed me the older state when I clicked the email link.